Sources/MongoSwift/MongoSwiftVersion.swift

```diff
@@ -1,6 +1,4 @@
 // Generated using Sourcery 0.16.1 — https://github.com/krzysztofzablocki/Sourcery
 // DO NOT EDIT
 
-
```

Comment:
did swiftlint start complaining about this suddenly?

Reply:
Yes, it was complaining about the vertical whitespace comments and saying they were superflous.